openshift/e2e-tests-openshift.sh

```diff
@@ -3,12 +3,7 @@
 set -e
 source $(dirname $0)/../vendor/github.com/tektoncd/plumbing/scripts/e2e-tests.sh
 source $(dirname $0)/resolve-yamls.sh
-if [[ "${1:-}" == "nightly" ]];then
-  source $(dirname $0)/pipeline-latest-release.sh
-else
-  source $(dirname $0)/pipeline-latest-release.sh "--only-stable-release"
-fi
```

Comment:
we might still need this part for versioned releases of triggers.

Reply:
This is not needed i have handled using if/else condition here https://github.com/openshift/tektoncd-triggers/pull/536/files#diff-8b42e7f32634a5db7922071ae21f9ec4a4ecd85c1ef603fd43bbc40faf35127bR18

Reply:
How will we handle versioned releases of triggers if only nightly releases are available?